frontend/views/containers/proposals/ProposalBox.vue

```diff
@@ -171,7 +171,7 @@ export default {
   grid-area: main;
   margin-top: 1.5rem;
   word-break: break-word;
-  min-width: 0; // So ellipsis work correctly inside grid.
```

Comment:
Unless the comment is saying something incorrect let's keep it around in case it's helpful to others

Reply:
it has been added back!